apps/examples/react-ssr-example/tsconfig.json

```diff
@@ -12,12 +12,5 @@
         "@server/*": ["src/server/*"],
         "@shared/*": ["src/shared/*"]
       }
-  },
-  "exclude": [
-      "./coverage",
-      "./dist",
-      "./docs",
-      "./node_modules",
-      "./stats"
-  ]
+  }
```

Comment:
Had to remove all these because it was bundling tests with the other dist files. So now we manage it all in one place.